ui/app/src/services/configuration.ts

```diff
@@ -95,6 +114,101 @@ export function getPreviewToolsConfig(site: string): Observable<PreviewToolsConf
   );
 }
 
+export function getAudiencesPanelConfig(site: string): Observable<ContentType> {
+  return getRawContent(site, `/targeting/targeting-config.xml`, 'studio').pipe(
+    map((content) => {
+      try {
+        return JSON.parse(content);
+      } catch (e) {
+        // Not JSON, assuming XML
+        let audiencesPanelContentType: ContentType = {
+          id: 'audiencesPanelConfig',
+          name: 'Audiences Panel Config',
+          type: 'unknown',
+          quickCreate: null,
+          quickCreatePath: null,
+          displayTemplate: null,
+          sections: null,
+          fields: null,
+          dataSources: null,
+          mergeStrategy: null
+        };
+
+        const xml = fromString(content);
+        const properties = Array.from(xml.querySelectorAll('property')).map((elem) => {
+
+          const //
+            name = getInnerHtml(elem.querySelector('name')),
+            label = getInnerHtml(elem.querySelector('label')),
+            type = getInnerHtml(elem.querySelector('type')),
+            defaultValue = getInnerHtml(elem.querySelector('default_value')),
+            hint = getInnerHtml(elem.querySelector('hint'));
+
+          let possibleValues: ContentTypeField['values'];
+
+          if (elem.querySelectorAll('value').length > 0) {
+            possibleValues = Array.from(elem.querySelectorAll('value')).map((element) => {
+              const value = getInnerHtml(element);
+              return {
+                label: element.getAttribute('label') ?? value,
+                value: value
+              }
+            });
+          } else {
+            possibleValues = [];
+          }
+
+          return {
+            id: name,
+            name: label,
+            type: audienceTypesMap[type],
+            sortable: null,
+            validations: null,
+            defaultValue,
+            required: null,
+            fields: null,
+            values: possibleValues,
+            helpText: hint
+          };
+
+        });
+
+        audiencesPanelContentType.fields = createLookupTable(properties, 'id');
+
+        return audiencesPanelContentType;
+      }
+    })
+  );
+}
+
+// TODO: asses the location of profile methods.
+export function fetchActiveModel(): Observable<ContentInstance> {
+  let audiencesPanelModel: ContentInstance;
```

Comment:
Remove this closure. You can return the objet so you don’t even need a var.
Move map operator to a new line inside pipe